twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -300,10 +298,14 @@ def _scrooge_scala_library_impl(ctx):
     all_java = java_common.merge(exports + [aspect_info.java_info])
   else:
     all_java = aspect_info.java_info
-  return [
-      DefaultInfo(files = aspect_info.output_files),
-      ScroogeInfo(aspect_info = aspect_info), all_java
-  ]
+
+  return struct(
+      scala = _create_scala_struct(ctx),  # For IntelliJ support
```

Comment:
We’re trying to kill this integration in favor of JavaInfo. Is there a specific issue this solves and which JavaInfo doesn’t solve?

Reply:
Sadly, yes. In order to use `JavaInfo` here, I need a way of setting multiple output jars on a `JavaInfo`, but neither the `JavaInfo` constructor nor the `java_common.create_provider` allow me to do this.

Once [this fix](https://github.com/bazelbuild/bazel/commit/4ff32da2f79f36fbb1e608830835a31f39772b8a) is released, we can ditch the `scala` field and go back to just returning a `JavaInfo` with multiple output jars.

Reply:
sounds good. WDYT about adding a TODO and an issue to track this?